ddtrace/settings/config.py

```diff
@@ -271,6 +271,7 @@ def __init__(self):
         self._trace_compute_stats = asbool(
             os.getenv("DD_TRACE_COMPUTE_STATS", os.getenv("DD_TRACE_STATS_COMPUTATION_ENABLED", in_gcp_function()))
         )
+        self._data_streams_enabled = asbool(os.getenv("DD_DATA_STREAMS_ENABLED"))
```

Comment:
Follow convention of other checks for envars and make a bit more obvious what default is?

```suggestion
        self._data_streams_enabled = asbool(os.getenv("DD_DATA_STREAMS_ENABLED", False))
```